recipes/leftbar/general_row/general_row.vue

```diff
@@ -25,7 +25,6 @@
           name="left"
         >
           <dt-recipe-leftbar-general-row-icon
-            :class="{ 'd-o50': isTyping }"
```

Comment:
Removing this class was part of design request too? 🤔

Reply:
Nah we just made a change a while back to not show the avatar at all rather than show it at 50% opacity. So this class would never be triggered.